test/unit/gateways/decidir_plus_test.rb

```diff
@@ -0,0 +1,155 @@
+require 'test_helper'
+
+class DecidirPlusTest < Test::Unit::TestCase
+  include CommStub
+
+  def setup
+    @gateway = DecidirPlusGateway.new(public_key: 'public_key', private_key: 'private_key')
+    @credit_card = credit_card
+    @amount = 100
+
+    @options = {
+      payment_id: '2bf7bffb-1257-4b45-8d42-42d090409b8a|448459',
+      site_transaction_id: SecureRandom.hex,
```

Comment:
This can be removed since the gateway does this as part of the purchase method now